src/mlpack/methods/lsh/lsh_search.hpp

```diff
@@ -174,6 +178,24 @@ class LSHSearch
   //! Get the second hash table.
   const arma::Mat<size_t>& SecondHashTable() const { return secondHashTable; }
 
+  //! Get the projection tables.
+  const std::vector<arma::mat> getProjectionTables() { return projections; }
```

Comment:
We should call this `Projections()`, not `getProjectionTables()`, like the rest of the accessors in mlpack: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#naming-conventions